spec/queries/homepage/articles_query_spec.rb

```diff
@@ -12,6 +12,12 @@
       expect(described_class.call.ids).to eq([article.id])
     end
 
+    it "does not return scheduled articles" do
+      scheduled_article = create(:article, published_at: Date.current + 5.days)
+
+      expect(described_class.call.ids).not_to eq([scheduled_article.id])
```

Comment:
```suggestion
      expect(described_class.call.ids).not_to include(scheduled_article.id)
```

I think the above suggestion is a more durable test.  For example, if the scheduled_article's ID is `1` and the returned IDs is `[1,2,3]`; then the original spec would pass.